models/models.go

```diff
@@ -0,0 +1,13 @@
+package models
```

Comment:
Since this only has the validation error response, it may be best to create a `responses` package and have this live there since it's a response and not a model.

Reply:
Thanks! I will make this change.